From 6bd2f0753d71395dcd3c91cacf49a2a59d011229 Mon Sep 17 00:00:00 2001
From: santhana-kumar <santhana-kumar@adlink.com>
Date: Thu, 2 Jan 2025 11:29:29 +0000
Subject: [PATCH] lec-imx8mp: modify soc.mak to compile lec-imx8mp.dtb

Signed-off-by: santhana-kumar <santhana-kumar@adlink.com>

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index 085224d..a2b4291 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -151,9 +151,9 @@ u-boot-atf-tee.bin: u-boot.bin bl31.bin $(TEE)
 clean:
 	@rm -f $(MKIMG) u-boot-atf.bin u-boot-atf-tee.bin u-boot-spl-ddr.bin u-boot.itb u-boot.its u-boot-ddr3l.itb u-boot-ddr3l.its u-boot-spl-ddr3l.bin u-boot-ddr4.itb u-boot-ddr4.its u-boot-spl-ddr4.bin u-boot-ddr4-evk.itb u-boot-ivt.itb u-boot-ddr4-evk.its $(OUTIMG)
 
-dtb = evk.dtb
+dtb = lec-imx8mp.dtb
 $(dtb):
-	./$(DTB_PREPROC) $(PLAT)-evk.dtb $(dtb) $(dtbs)
+	./$(DTB_PREPROC) lec-imx8mp.dtb $(dtb) $(dtbs)
 
 u-boot.itb: $(dtb) $(supp_dtbs)
 	./$(PAD_IMAGE) $(TEE)
-- 
2.34.1

